CTP/sql/bin/run.sh

```diff
@@ -204,6 +208,11 @@ function uniteName()
     fi
 }
 
+function clean_log_cores()
+{
+     rm -rf $CUBRID/logs/* 2>&1 > /dev/null
+     find $CUBRID/* -type -f -name "core*"|xargs -i rm {}
```

Comment:
1. 
as is： -type -f
to be:  -type f

2. find both $CUBRID and $CTP_HOME directories.